lib/devicetrust/native/device_darwin.go

```diff
@@ -142,3 +145,7 @@ func getDeviceCredential() (*devicepb.DeviceCredential, error) {
 func statusErrorFromC(res C.int32_t) error {
 	return &statusError{status: int32(res)}
 }
+
+func solveTPMEnrollChallenge(challenge *devicepb.TPMEnrollChallenge) (*devicepb.TPMEnrollChallengeResponse, error) {
+	return nil, devicetrust.ErrPlatformNotSupported
```

Comment:
Maybe:

```suggestion
	return nil, trace.BadParameter("called solveTPMEnrollChallenge on darwin")
```

so we don't mix programming errors with the more "banal" ErrPlatformNotSupported?

I considered trace.NotImplemented too, but that often has special handling so I'd rather not risk it either.